HIP/hip-0000-event-serialization.md

```diff
@@ -0,0 +1,227 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Event Serialization With Round Number, EventDescriptors, and Multiple Other Parents.
+author: Edward Wertz <edward@swirldslabs.com>
+working-group: Swirlds Labs Platform Hashgraph Team
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: 2023-10-25
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/827
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
+---
+
+## Abstract
+
+This HIP proposes several changes to the event serialization format.
+
+1. Add the round during which an event was created to the event's meta-data.
```

Comment:
This is confusing, because it makes me think of round created, which is assigned by consensus, but this is not that round.

Reply:
I have modified the wording to: 
```
 Add the number of the latest round to come to consensus to the metadata of new events.  
This number is used to  look up the correct consensus roster for validating the event's signature. 
```